cdap-app-templates/cdap-etl/cdap-etl-lib/src/main/java/co/cask/cdap/template/etl/common/BatchFileFilter.java

```diff
@@ -52,54 +52,59 @@
 public class BatchFileFilter extends Configured implements PathFilter {
 
   private static final Logger LOG = LoggerFactory.getLogger(FileBatchSource.class);
-  //length of 'YYYY-MM-dd-HH-mm"
-  private static final int DATE_LENGTH = 16;
+  //length of 'YYYY-MM-dd-HH"
+  private static final int DATE_LENGTH = 13;
```

Comment:
can we just derive this from either the pattern string directly, or `sdf.toPattern().length()`, so we don't need to compute this manually